### DISCUSSION TITLE
Is there any software that can batch translate English subtitles to bilingual subtitles (original on top, translation on bottom) via deepl?

### DISCUSSION THREAD

**Discussion by Skylarzz** [SubtitleEdit/subtitleedit#8830] [Q&A]
Now I can only translate them one by one with SubtitleEdit GUI

**Comment by realtechspecs** [comment]
> Now I can only translate them one by one with SubtitleEdit GUI

Do you want both the original and the translated subtitles to be included within a single, self-contained SRT file?

**Reply by Skylarzz** [reply]
You're right, it is like that.
